alembic/testing/suite/test_autogen_diffs.py

```diff
@@ -18,7 +18,7 @@ class AlterColumnTest(AutogenFixtureTest, TestBase):
 
     @testing.combinations((True,), (False,))
     @config.requirements.comments
-    def test_all_existings_filled(self, pk):
+    def test_all_existing_filled(self, pk):
```

Comment:
i don't think the extraneous `s` adds value here.

Reply:
These tests can be overloaded by extensions. While unlikely maybe we could revert these changes for this file?